### PR TITLE
Add an option to show Arabic tafseer in Quran translation and fixed a minor bug

### PR DESCRIPTION
**Pull request by murtuzaalisurti**
Successfully added an option which when selected, displays the tafseer in Arabic. Also, fixed a minor bug which wasn't displaying a translation language option correctly!

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -240,10 +240,14 @@ $(document).ready(function(){
...
-                  } else if(data.data[i].edition.language === "ar"){
+                  } else if(data.data[i].edition.language === "ar" && data.data[i].edition.identifier !== "ar.muyassar"){
                     setVerse(data.data[i]);
                     verse.data = data.data[i];
-                  } else {
+                  } else if(data.data[i].edition.language === "ar" && data.data[i].edition.identifier === "ar.muyassar"){
+                    verse.translation = data.data[i];
+                    setTranslation(data.data[i]);
+                  } 
+                  else {
```
Comment (shahednasser) [inline js/main.js:250]: I think it isn't necessary to make a special case for Arabic language anymore, as it is now allowed to have translation.

However, we need to apply two styling properties on the `.translation-container .body` element when the translation is for Arabic:

```css
font-size: 17px;
direction: rtl;
```

You can do it in the `setTranslation` function by passing it the language as a second parameter. Then, you can place the above styles in a class (for example `.ar-translation`) and in `setTranslation` check if the language is `ar` then the class should be added to `.translation-container .body`.

If you also have a better approach then please go ahead.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @murtuzaalisurti for your work I just have a comment below

File: js/main.js (hunk 1 above)
Comment (murtuzaalisurti) [reply js/main.js]: did required changes! https://github.com/shahednasser/quran-extension/pull/21/commits/3371f1650cb8109e4c611452a54021de2dba9e40

File: js/main.js [hunk 2]
```diff
@@ -240,12 +240,13 @@ $(document).ready(function(){
                 for(let i = 0; i < data.data.length; i++){
                   if(data.data[i].hasOwnProperty('audio')){
                     verse.audio = data.data[i].audio;
-                  } else if(data.data[i].edition.language === "ar"){
+                  } else if(data.data[i].edition.type === "quran"){
```
Comment (shahednasser) [inline js/main.js:243]: Is this condition necessary?

Comment (murtuzaalisurti) [reply js/main.js:243]: the endpoint returns 3 objects, the first one is the verse, the second is for audio and the third one is for translation/tafsir.

**Comment by shahednasser** [comment]
Great thank you so much!

**Pull request merged**